docs: Fix user documentation

synicalsyntax opened the pull request:
**Summary:**

* Remove **Chat with Zulip button** doc; the images referenced in this doc have been
deleted for some reason, rendering it almost useless with the missing
images.

* Fix formatting errors in various docs.

* Change wording in various docs for better clarification.

* Conform old and new docs to documentation styling guidelines.

smarx commented:
*Automated message from Dropbox CLA bot*

@synicalsyntax, it looks like you've already signed the Dropbox CLA. Thanks!

timabbott commented:
For the "Chat with Zulip" button one; we should probably not delete it but instead fix it.

timabbott commented:
Thanks for working on this @synicalsyntax!  

I merged everything but removing the "Chat with Zulip" button and `docs: Update *Verify that you message has been successfully sent* doc.`.  For the latter, I think you misunderstood what the previous version was saying in:

```
-Zulip is designed to store locally echoed message content in local
-storage and replay it when your browser reconnects to the Internet, to
-avoid issues where messages users thought they sent are lost.
```

We don't expect most users to be watching for the subtle missing-timestamp property; that is primarily to make it possible to debug visually whether a message has been sent (and have something that users who are carefully inspecting a message can identify).  The thing to protect users from losing messages is that we store them in browser local storage and attempt to retransmit them when the client is back online.  This feature seems to be not working properly right now (and will probably be transmuted into an integration with the "drafts" feature), but we'll probably fix it soon.  The new framing makes it sound like the timestamp thing is there to prevent users from losing messages (and you're supposed to be on the lookout for this as a user!), which is an unreasonable thing to be expecting users to do and makes the doc feel a bit out of touch.  

You should be able to just rebase your branch (which will automatically drop all but the 2 remaining commits), fix those 2 commits up, and then ping me on this PR to merge the cleaned up versions of the rest.  Thanks @synicalsyntax!

synicalsyntax commented:
@timabbott I've fixed the **Join Zulip Chat button** doc, but I'm unsure if I fixed what you requested in `docs: Update *Verify that you message has been successfully sent* doc.` Can you please review it to see if your requested changes were fixed?
